### PR TITLE
[www] Update the Install docs to use Personal Access Tokens

### PR DESCRIPTION
**Pull request by cnunciato**
Also removes some dead code.

Signed-off-by: Christian Nunciato <chris@nunciato.org>

tenor-36484621

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: www/source/partials/docs/_using-hab-config-workstation.html.md.erb [hunk 1]
```diff
@@ -3,9 +3,9 @@ Once Habitat has been installed, the <code>hab</code> CLI makes it easy to get y
 
 <img alt="screenshot of hab setup output in CLI" src="/images/screenshots/hab-setup.png">
 
-Setup asks you to create a new origin and a set of origin keys. Optionally, you can also provide a [GitHub personal access token](https://help.github.com/articles/creating-an-access-token-for-command-line-use/) to upload packages to the public depot and share them with the Habitat community.
+Setup asks you to create a new origin and a set of origin keys.
 
-> The GitHub personal access token needs information provided from the <code>user:email</code> and <code>read:org</code> OAuth scopes. Habitat uses the information provided through these scopes for authentication and to determine features based on team membership.
+Optionally, you can also provide a Habitat personal access token to upload packages to the public depot and share them with the Habitat community. To obtain a personal access token, [visit your Profile page](<%= builder_web_url %>/#/profile) on the Builder website and follow the instructions; you can apply the token using the `HAB_AUTH_TOKEN` environment variable so that any commands requiring authentication will use it.
```
Comment (brewn) [inline www/source/partials/docs/_using-hab-config-workstation.html.md.erb]: I'd like to simplify this by just linking them to the access token section of the builder guide: https://www.habitat.sh/docs/using-builder/#builder-token

**Comment by cnunciato** [comment]
@brewn Thanks for that commit! I'm going to go ahead and merge this, then.

**Comment by cnunciato** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/354448227).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/354448227).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**